RELEASE_NOTES.md

```diff
@@ -1,8 +1,12 @@
 Migration Instructions
 ======================
 
-* The `Version` alias to `versionInfo` provided by the `Version` module has been removed.
-  All remaining usage can be replaced with `versionInfo`.
+* The AA containing all the automatically deduced version information was
+  earlier called `versionInfo` (provided by the `Version` module). This name is
+  now deprecated. All usage of this name should be replaced with `version_info`.
+
+* The `Version` alias to the now deprecated `versionInfo` has been removed. All
+  remaining usage should be replaced with `version_info`.
```

Comment:
Normally deprecations go into their own section. But I guess this will have to be changed when rebased to v1.x.x anyway.